configs/nextflow.hopper.config

```diff
@@ -44,17 +43,22 @@ params {
   rCRS_fasta        = "${refpath}/fasta/split/M.fa"
 
   // VEP REFERENCES AND ANNOTATION DBS //
+  VEP_DIR           = "${refpath}/vep/.vep111.0"
+  VEP_CACHE         = "${VEP_DIR}"
+  VEP_PLUGINS       = "${VEP_DIR}/Plugins"
+  VEP_SYNONYMS      = "${VEP_DIR}/synonyms"
+  VEP_FASTA         = "${VEP_DIR}/homo_sapiens_merged/111_GRCh38/Homo_sapiens.GRCh38.dna.toplevel.fa.gz"
+  VEP_MAX_SV_SIZE   = "50000000"
```

Comment:
Are these two the default values?

Reply:
nyet. MAX_SV_SIZE is 5k. distance doesn't say.